### PR TITLE
nss: allow to specify TLS 1.3 ciphers if supported by NSS

### PR DESCRIPTION
**Pull request by kdudka**

### CODE REVIEW

File: lib/vtls/nss.c [hunk 1]
```diff
@@ -216,6 +216,11 @@ static const cipher_s cipherlist[] = {
  {"dhe_rsa_chacha20_poly1305_sha_256",
      TLS_DHE_RSA_WITH_CHACHA20_POLY1305_SHA256},
 #endif
+#ifdef TLS_AES_256_GCM_SHA384
```
Comment (jay) [inline lib/vtls/nss.c]: How did you come up with this as opposed to TLS_CHACHA20_POLY1305_SHA256? In other words if TLS_AES_256_GCM_SHA384 is defined then TLS_CHACHA20_POLY1305_SHA256 is definitely defined?

Comment (kdudka) [reply lib/vtls/nss.c]: > How did you come up with this as opposed to TLS_CHACHA20_POLY1305_SHA256?

I did not, @tomato42 did.

> In other words if `TLS_AES_256_GCM_SHA384` is defined then `TLS_CHACHA20_POLY1305_SHA256` is definitely defined?

I think so.  All the defines were introduced in a single commit:

https://github.com/nss-dev/nss/commit/e272467e#diff-61a352311f838d5a89d178abc1f4bb6aR207

Comment (jay) [reply lib/vtls/nss.c]: Ok. Please add them to the [NSS section of CIPHERS.md](https://github.com/curl/curl/blob/curl-7_65_0/docs/CIPHERS.md#nss). Are they set using CURLOPT_TLS13_CIPHERS? OpenSSL CIPHERS has a [separate section](https://github.com/curl/curl/blob/curl-7_65_0/docs/CIPHERS.md#tls-13-cipher-suites) for TLS 1.3 ciphers, is something similar is needed for NSS?

Comment (tomato42) [reply lib/vtls/nss.c]: OpenSSL uses separate API for setting TLS 1.3 ciphers, NSS doesn't

**Comment by kdudka** [comment]
Thanks for review!  Merging...

**Pull request closed without merging**